shared/commonprops/commonProps.ts

```diff
@@ -21,11 +29,29 @@ export async function downloadModel() {
         var result = await init(resp + '/model.tflite', false);
         console.log('model download result is = ' + result);
         if (result) {
+          sendImpressionEvent(
+            getImpressionEventData(
+              TelemetryConstants.FlowType.faceModelInit,
+              TelemetryConstants.EndEventStatus.success,
+            ),
+          );
           break;
         }
+        sendErrorEvent(
+          getErrorEventData(
+            TelemetryConstants.FlowType.faceModelInit,
+            TelemetryConstants.EndEventStatus.failure,
```

Comment:
This should the landing screen's name instead of the status. If you want to pass the status send it in additional parameters